examples/platform/silabs/SiWx917/SiWx917/sl_wlan_config.h

```diff
@@ -48,21 +48,11 @@ static const sl_wifi_device_configuration_t config = {
 #endif
                                                 | SL_SI91X_TCP_IP_FEAT_ICMP | SL_SI91X_TCP_IP_FEAT_EXTENSION_VALID),
                      .custom_feature_bit_map     = (SL_SI91X_CUSTOM_FEAT_EXTENTION_VALID | RSI_CUSTOM_FEATURE_BIT_MAP),
-                     .ext_custom_feature_bit_map = (
-#ifdef SLI_SI917
-                         (RSI_EXT_CUSTOM_FEATURE_BIT_MAP)
-#else // defaults
-#ifdef SLI_SI91X_MCU_INTERFACE
```

Comment:
We are loosing this `ifdef` is that intended ?

Reply:
Yes since this` #ifdef ` was in the `#else`, which we would never end up in since `SLI_SI917 ` will always be defined as it is needed by the wifi_sdk